_credentials/equinet.js

```diff
@@ -0,0 +1,38 @@
+
+import { v4 as uuidv4 } from "uuid";
+import { getRandomNumber } from "utils";
+
+export function createCreditScoreCredential({ receiverDid }) {
+  const dockUrl = "https://api-testnet.dock.io";
```

Comment:
We should use the env variable here

Reply:
```suggestion
  const dockUrl = process.env.DOCK_API_URL;
```

Reply:
This change is ok but please commit them and mark the conversation as resolved after so it's easier to track.

Reply:
Fixed dockUrl around the application to use .env.